readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -152,6 +152,484 @@ def test_new_tag_update_inactive(self):
         version_8 = Version.objects.get(slug='0.8.3')
         self.assertFalse(version_8.active)
 
+    def test_normal_behavior_for_stable_after_deleting_user_defined_tag(self):
```

Comment:
Can you use `pytest.mark.xfail(strict=True)` here? That way, these test will fail and pytest won't fail but pytest will fail when you write the logic and they start passing.